src/coreclr/inc/yieldprocessornormalized.h

```diff
@@ -20,6 +18,26 @@ FORCEINLINE void System_YieldProcessor() { YieldProcessor(); }
     T() = delete; \
     DISABLE_COPY(T)
 
+#ifdef FEATURE_NATIVEAOT
+#define SIZE_T uintptr_t
+typedef BYTE UINT8;
+typedef ULONGLONG UINT64;
```

Comment:
Could you please convert the code that uses these types to use standard C/C++ types (`uintptr_t`, `size_t`, `uint8_t`, etc.) instead of adding these typedefs.